Update clases.py

giulianaleoni opened the pull request:
Cambie lista por diccionario para buscar mejor

giulianaleoni committed:
Update clases.py

Cambie lista por diccionario para buscar mejor

zLucas2021 commented on desafio8grupo1/clases.py:
```diff
@@ -1,7 +1,7 @@
 
 class Usuario:
-    usuarios_registrados = []
+    usuarios_registrados = {}
```
esta bien que sea un diccionario , pero tenes que fijarte que por cada instancia de usuario , creas un diccionario,  lo mejor seria que lo saques afuera de la clase , y si queres tambien podria ser una lista , ya que en el login vos no devolves nada de ese usuario

zLucas2021 commented on desafio8grupo1/clases.py:
```diff
@@ -14,13 +14,26 @@ def __init__(self, id, nombre, apellido, telefono, username, email, contraseña,
 
-def registrar(self, id, nombre, apellido, telefono, username, email, contraseña, fecha_registro, avatar):
+    def login(self, username, contraseña):
+        if username in self.usuarios_registrados:
```
no creo que este bien la condicion , lo tendria que probar

zLucas2021 commented on desafio8grupo1/clases.py:
```diff
@@ -14,13 +14,26 @@ def __init__(self, id, nombre, apellido, telefono, username, email, contraseña,
-def registrar(self, id, nombre, apellido, telefono, username, email, contraseña, fecha_registro, avatar):
+    def login(self, username, contraseña):
+        if username in self.usuarios_registrados:
+            usuario = self.usuarios_registrados[username]
```
y en el caso de ser un dic , solo te trae un solo elemento

zLucas2021 commented on desafio8grupo1/clases.py:
```diff
@@ -14,13 +14,26 @@ def __init__(self, id, nombre, apellido, telefono, username, email, contraseña,
+                self.estado = usuario.estado
+                self.online = usuario.online
+                print("Inicio de sesión exitoso.")
+                return
```
return True

zLucas2021 commented on desafio8grupo1/clases.py:
```diff
@@ -1,7 +1,7 @@
 class Usuario:
-    usuarios_registrados = []
+    usuarios_registrados = {}
     def __init__(self, id, nombre, apellido, telefono, username, email, contraseña, fecha_registro, avatar, estado, online):
```
Aca habria que ver como se instancia , ya que tendrias que cargar valores nulos

giulianaleoni committed:
Update clases.py

15/16 cambie los parametros como puso el profe hoy en la clase.

y para el login volvimos a la lista a ver si ahi se puede

giulianaleoni committed:
Update clases.py
